lib/model/model/mock/MockStage.dart

```diff
@@ -18,42 +18,57 @@ class MockStage extends MockEntity<NewStageEntity> {
   MockStage({int seed}) : this._rand = Random(seed);
 
   NewStageEntity build() {
-      final start = _mockDate.randomMonthAgo();
-      return NewStageEntity(id: _identifiers.identifier(), article: _articleBuilder.buildStage(), started: start, ended: null);
+    final start = _mockDate.randomMonthAgo();
+    return NewStageEntity(
+        id: _identifiers.identifier(),
+        article: _articleBuilder.buildStage(),
+        started: start,
+        ended: null);
   }
 
-  NewStageEntity buildNewFromArticle(ArticleEntity article){
-    return NewStageEntity(id: _identifiers.identifier(), article: article);
+  NewStageEntity buildNewFromArticle(ArticleEntity article) {
+    return NewStageEntity(
+      id: _identifiers.identifier(),
+      article: _articleBuilder.buildStageFromArticle(article),
```

Comment:
Instead of passing the article directly, using `_articleBuilder` to create an Article suited for Stages (with no images)